javascript.js

```diff
@@ -0,0 +1,20 @@
+// set variavles
+var user = []; //inline coment
+
+/**
```

Comment:
Typo inside comment in word `togeter`. Possible corrections: together (43), toaster (0),